blocks/identity-block/utils/useSales.js

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
```

Comment:
useIdentity was defined inside [arc-themes-component ](https://github.com/WPMedia/arc-themes-components/tree/arc-themes-release-version-2.2.0/src/utils/hooks/use-identity) project. I think it's worth defining this hook there too.

Reply:
https://github.com/WPMedia/arc-themes-components/pull/242

Reply:
@edwardcho1231 Now It can be removed